scripts/azureml-assets/azureml/assets/release/asset_publish.py

```diff
@@ -95,13 +94,16 @@ def model_prepare(
             update_spec_file(spec_file, model_dir)
 
     elif model_config.type == assets.ModelType.MLFLOW:
-        mlflow_utils = MLFlowModelUtils(
-            name=asset.name,
-            task_name=model_config.task_name.value,
-            flavor=model_config.flavor.value,
-            mlflow_model_dir=model_dir,
+        # TODO: udpate this once we start consuming git based model
+        model_utils = ModelUtils(
```

Comment:
>ModelUtils [](http://example.com/codeflow?start=22&length=10)

Could we use `ModelConfig` or something similar?

Reply:
Only if it's passed as an arg to functions in model_utils please, since only want YAML handling code in ModelConfig.

Reply:
we have modelConfig. Made ModelUtils class static